### DISCUSSION TITLE
Error in deconvolution with cibersort

### DISCUSSION THREAD

**Discussion by ericrabbit** [omnideconv/immunedeconv#85] [Q&A]
>>> Running cibersort
Error in (function (sig_matrix = lm22, mixture_file, perm, QN = TRUE,  : 
  None identical gene between eset and reference had been found.
         Check your eset using: intersect(rownames(eset), rownames(reference))

Using vlookup() in excel can find 20 genes in the LM22 which cannot be found in my expression matrix.
What can I do next?

**Comment by grst** [comment]
While not ideal, *some* missing genes should not be a problem. In this case it can't find any. If you are running immunedeconv with an `eSet`, did you specify the column that contains the gene symbols correctly? 

What does your call to `deconvolute` look like?
